index.js

```diff
@@ -163,6 +139,84 @@ function processDuplicateFree(/* CODE HERE ONLY AFTER COMPLETING ALL OTHER TASKS
 /////////////// HIGHER-ORDER ARRAY METHODS ///////////////
 /////////////// HIGHER-ORDER ARRAY METHODS ///////////////
 
+
+/**
+ * ### Challenge `lowerCaseStrings`
+ * 
+ * @instructions
+ * Implement this function using forEach().
+ * 
+ * @param strings an array of strings.
+ * @returns an array of equal length to `strings` containing lowercased versions of each string.
+ * 
+ * 
+ * Examples of usage of this function:
+ * [1] Invoking `lowerCaseStrings` with `[ 'Orange', 'APPLE', 'banana', 'mAnGo']` will return `[ 'orange', 'apple', 'banana', 'mango' ]`.
+ * 
+ * [2] Invoking `lowerCaseStrings` with `['a', 'b', 'c' ]` will return `[ 'a', 'b', 'c' ]`.
+*/
+function lowerCaseStrings(/* code here */) {
+  /* code here */
+}
+
+/**
+ * ### Challenge `isItAnApple`
```

Comment:
Isn't this the same exercise "isItAnApple" from the JS-Exercise-Functions-Arrays-Objects challenge?

Reply:
It's the same function, but they're being asked to implement it with .map() this time instead of with a for loop.

Reply:
Ohhh, nice